user-interface/src/main/java/life/qbic/datamanager/Application.java

```diff
@@ -88,6 +95,23 @@ private static void setupUseCases(ConfigurableApplicationContext context) {
     newPassword.setUseCaseOutput(newPasswordHandler);
   }
 
+  private static MessageSubscriber whenUserRegisteredAddSid(SidRepository sidRepository) {
```

Comment:
Please add another TODO statement here, so we really address it. The Application.java class must not contain policies in the future to avoid making this class a spaghetti script of initialization.